test/e2e/storage/testsuites/provisioning.go

```diff
@@ -265,6 +221,26 @@ func TestDynamicProvisioning(t StorageClassTest, client clientset.Interface, cla
 		Expect(err).NotTo(HaveOccurred())
 	}
 
+	// Determine where to run pods.
+	nodeName := t.NodeName
```

Comment:
is this a framework change or a test change? It's unclear to me if this belongs in the same PR

Reply:
Same problem: this change is from PR #72002